cmd/cmd.go

```diff
@@ -49,6 +51,7 @@ var (
 	targetFolder string
 )
 
+// AddFlags adds flags to the provided command.
```

Comment:
does not provide any interesting info.

for example: 
If the comment for `fmt.Println` would be `Println prints the string provided to stdout`
- Don't know about you, but this is not very informative

the actual comment on `fmt.Println`:
```
// Println formats using the default formats for its operands and writes to standard output.
// Spaces are always added between operands and a newline is appended.
// It returns the number of bytes written and any write error encountered.
```
- this tells you things that are not necessarily obvious